src/main/webapp/find-events.html

```diff
@@ -39,5 +39,49 @@ <h1>find events</h1>
         </div>
       </div>
     </div>
+
+    <div id="modal-background">
+      <div id="modal-create-post">
+        <button id="modal-close" type="button">&#10005;</button>
+        <h1 id="modal-title">Add a Post</h1>
```

Comment:
Since the button is called `create a post`, should the modal title be `create a post` as well?

Reply:
Good point, Ill change it